contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1652,6 +1653,17 @@ define_escape_hatch_variables(void)
 							 PGC_USERSET,
 							 GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
 							 NULL, NULL, NULL);
+	
+	/* INLINE option */
+	DefineCustomEnumVariable("babelfishpg_tsql.escape_hatch_inline_function",
```

Comment:
NIT: I think `escape_hatch_inline_function_option` should be better name for this.